cli/Cargo.toml

```diff
@@ -16,7 +16,7 @@ futures = "0.3"
 clap = { version = "3.0", features = ["derive", "env"] }
 open = { version = "2.1.0" }
 reqwest = { version = "0.11.9", default-features = false, features = ["json", "stream", "native-tls-vendored"] }
-tokio = { version = "1.23", features = ["full"] }
+tokio = { version = "1.24.2", features = ["full"] }
```

Comment:
Seems like the lock file picked up more recent version `1.25.0`, should this instead be bumped to the same ?